api/server/annotations.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+package server;
+
+//option (errors_swagger) = {
+//  responses: {
+//    key: "default"
+//    value: {
+//      description: "An error response."
+//      schema: {
+//        json_schema: {
+//          type: STRING
+//        }
+//      }
+//    }
+//  }
+//};
+option go_package = "server";
+
+import "google/api/annotations.proto";
+import "google/protobuf/descriptor.proto";
+import "vendor/github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger/options/openapiv2.proto";
+
+// ErrorResponse is a message returned on HTTP error.
+message ErrorResponse {
+  string error = 1;
+  string message = 2;
+  int32 code = 3;
+}
```

Comment:
Could we use that type in `assert` in tests?

Reply:
because of using flag `--with-flatten=expand` in flatten we can't. If we don't use it, we will have to make a lot of changes in code, because it generates new types for all responses..